build_tools/docker/dockerfiles/nvidia-bleeding-edge.Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# An image for building IREE using CMake and testing IREE with NVIDIA GPUs.
+
+# To use the host GPUs, `docker run` must be called with the `--gpus all` flag.
+
+# We use .deb files that we host because we have to pin the version exactly to
```

Comment:
Huh like none of these comments should be true anymore. I added the Nvidia container toolkit to our VM images, so we don't need the matching version stuff. I do think that mirroring and pulling form our own storage is still maybe a good idea. But there are especially comments betlow about Ubuntu 18.04 that don't make much sense given that this is 22.04